doc/source/reshaping.rst

```diff
@@ -93,6 +92,12 @@ You can then select subsets from the pivoted ``DataFrame``:
 Note that this returns a view on the underlying data in the case where the data
 are homogeneously-typed.
 
+.. note::
+   ``pandas.pivot`` will error with a ``ValueError: Index contains duplicate
```

Comment:
Can we convert `pandas.pivot` into an inline reference, similar to how you have pandas.pivot_table?